tests/ssl_example_cert/cert.pem

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
The shell actions that were used to make these test files, I think, should be saved to this folder in an `.sh` file too.